docs/swagger.yaml

```diff
@@ -757,19 +758,42 @@ paths:
               $ref: '#/definitions/DetailedTag'
         500:
           description: Unexpected internal errors.
-  /repositories/manifests:
+    delete:
```

Comment:
There is another API DELETE /api/repositories/${name}/tags/${tag}, so DELETE /api/repositories/${name}/tags/ may be make more sense?